Classes/Charts/SUIChartView/SUIChartView.swift

```diff
@@ -0,0 +1,89 @@
+// Copyright 2025 Pera Wallet, LDA
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//    http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//   SUIChartView.swift
+
+import SwiftUI
+import Charts
+
+struct ChartDataPoint: Identifiable, Hashable {
+    let id = UUID()
+    let day: String
+    let value: Double
+}
+
+struct SUIChartView: View {
```

Comment:
Initials prefixes is something that was used in the obj-c era. Please don't use it anymore.